<?xml version="1.0" encoding="utf-8" ?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
        "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="" lang="">
<head>
  <title>Test Results Report - RUN ON FIREFOX_UI</title>
  <meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
  <meta name="description" content="TestNG unit test results." />
  <link href="reportng.css" rel="stylesheet" type="text/css" />
    <script type="text/javascript" src="reportng.js"></script>
</head>
<body>
<h1>RUN ON FIREFOX_UI</h1>
<p>
  Test duration: 43.686s
</p>



  <table class="resultsTable">
    <tr><th colspan="3" class="header failed">Failed Tests</th></tr>
          <tr>
        <td colspan="3" class="group">com.nopcommerce.admin.Level_11_Demo_Verify</td>
      </tr>
            
<tr>
  <td class="method">
                  TC_01_Register
      </td>
  <td class="duration">
    9.074s
  </td>
  <td class="result">
        
    
            <div class="testOutput">
                        [INFO ] [2021-06-03 11:04:10,352] mo trang nopcommerce<br/>
&lt;br&gt;<br />
                                [INFO ] [2021-06-03 11:04:10,375] click to register link<br/>
&lt;br&gt;<br />
                                [INFO ] [2021-06-03 11:04:17,118]&nbsp; -------------------------- FAILED -------------------------- <br/>
&lt;br&gt;<br />
                  </div>
    

        <a href="javascript:toggleElement('exception-0', 'block')" title="Click to expand/collapse"><b>java.lang.AssertionError: expected [Your registration completed...] but found [Your registration completed]</b></a><br />
    <div class="stackTrace" id="exception-0">
              org.testng.Assert.fail(Assert.java:96)<br />
              org.testng.Assert.failNotEquals(Assert.java:776)<br />
              org.testng.Assert.assertEqualsImpl(Assert.java:137)<br />
              org.testng.Assert.assertEquals(Assert.java:118)<br />
              org.testng.Assert.assertEquals(Assert.java:442)<br />
              commons.AbstractTest.checkEquals(AbstractTest.java:142)<br />
              commons.AbstractTest.verifyEquals(AbstractTest.java:154)<br />
              com.nopcommerce.admin.Level_11_Demo_Verify.TC_01_Register(Level_11_Demo_Verify.java:75)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)<br />
              sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)<br />
              java.lang.reflect.Method.invoke(Unknown Source)<br />
              org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)<br />
              org.testng.internal.Invoker.invokeMethod(Invoker.java:583)<br />
              org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)<br />
              org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)<br />
              org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)<br />
              org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)<br />
              org.testng.TestRunner.privateRun(TestRunner.java:648)<br />
              org.testng.TestRunner.run(TestRunner.java:505)<br />
              org.testng.SuiteRunner.runTest(SuiteRunner.java:455)<br />
              org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)<br />
              org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)<br />
              org.testng.SuiteRunner.run(SuiteRunner.java:364)<br />
              org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)<br />
              org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)<br />
              org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)<br />
              org.testng.TestNG.runSuitesLocally(TestNG.java:1137)<br />
              org.testng.TestNG.runSuites(TestNG.java:1049)<br />
              org.testng.TestNG.run(TestNG.java:1017)<br />
              org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)<br />
              org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)<br />
              org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)<br />
                </div>
      </td>
</tr>
<tr>
  <td class="method">
                  TC_02_Login
      </td>
  <td class="duration">
    23.493s
  </td>
  <td class="result">
        
    
            <div class="testOutput">
                        [INFO ] [2021-06-03 11:04:22,590]&nbsp; -------------------------- FAILED -------------------------- <br/>
&lt;br&gt;<br />
                                [INFO ] [2021-06-03 11:04:22,635]&nbsp; -------------------------- FAILED -------------------------- <br/>
&lt;br&gt;<br />
                                [INFO ] [2021-06-03 11:04:32,794]&nbsp; -------------------------- FAILED -------------------------- <br/>
&lt;br&gt;<br />
                                [INFO ] [2021-06-03 11:04:42,916]&nbsp; -------------------------- PASSED -------------------------- <br/>
&lt;br&gt;<br />
                  </div>
    

        <a href="javascript:toggleElement('exception-1', 'block')" title="Click to expand/collapse"><b>java.lang.Throwable: Multiple failures (3):<br/>
Failure 1 of 3<br/>
java.lang.AssertionError: expected [false] but found [true]<br/>
	at org.testng.Assert.fail(Assert.java:96)<br/>
	at org.testng.Assert.failNotEquals(Assert.java:776)<br/>
	at org.testng.Assert.assertFalse(Assert.java:65)<br/>
	at org.testng.Assert.assertFalse(Assert.java:75)<br/>
	at commons.AbstractTest.checkFailed(AbstractTest.java:126)<br/>
	at commons.AbstractTest.verifyFalse(AbstractTest.java:136)<br/>
	at com.nopcommerce.admin.Level_11_Demo_Verify.TC_02_Login(Level_11_Demo_Verify.java:89)<br/>
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br/>
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)<br/>
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)<br/>
	at java.lang.reflect.Method.invoke(Unknown Source)<br/>
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)<br/>
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)<br/>
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)<br/>
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)<br/>
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)<br/>
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)<br/>
	at org.testng.TestRunner.privateRun(TestRunner.java:648)<br/>
	at org.testng.TestRunner.run(TestRunner.java:505)<br/>
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)<br/>
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)<br/>
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)<br/>
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)<br/>
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)<br/>
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)<br/>
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)<br/>
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)<br/>
	at org.testng.TestNG.runSuites(TestNG.java:1049)<br/>
	at org.testng.TestNG.run(TestNG.java:1017)<br/>
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)<br/>
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)<br/>
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)<br/>
<br/>
Failure 2 of 3<br/>
java.lang.AssertionError: expected [false] but found [true]<br/>
	at org.testng.Assert.fail(Assert.java:96)<br/>
	at org.testng.Assert.failNotEquals(Assert.java:776)<br/>
	at org.testng.Assert.assertFalse(Assert.java:65)<br/>
	at org.testng.Assert.assertFalse(Assert.java:75)<br/>
	at commons.AbstractTest.checkFailed(AbstractTest.java:126)<br/>
	at commons.AbstractTest.verifyFalse(AbstractTest.java:136)<br/>
	at com.nopcommerce.admin.Level_11_Demo_Verify.TC_02_Login(Level_11_Demo_Verify.java:90)<br/>
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br/>
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)<br/>
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)<br/>
	at java.lang.reflect.Method.invoke(Unknown Source)<br/>
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)<br/>
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)<br/>
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)<br/>
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)<br/>
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)<br/>
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)<br/>
	at org.testng.TestRunner.privateRun(TestRunner.java:648)<br/>
	at org.testng.TestRunner.run(TestRunner.java:505)<br/>
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)<br/>
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)<br/>
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)<br/>
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)<br/>
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)<br/>
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)<br/>
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)<br/>
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)<br/>
	at org.testng.TestNG.runSuites(TestNG.java:1049)<br/>
	at org.testng.TestNG.run(TestNG.java:1017)<br/>
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)<br/>
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)<br/>
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)<br/>
<br/>
Failure 3 of 3<br/>
java.lang.AssertionError: expected [false] but found [true]</b></a><br />
    <div class="stackTrace" id="exception-1">
              org.testng.Assert.fail(Assert.java:96)<br />
              org.testng.Assert.failNotEquals(Assert.java:776)<br />
              org.testng.Assert.assertFalse(Assert.java:65)<br />
              org.testng.Assert.assertFalse(Assert.java:75)<br />
              commons.AbstractTest.checkFailed(AbstractTest.java:126)<br />
              commons.AbstractTest.verifyFalse(AbstractTest.java:136)<br />
              com.nopcommerce.admin.Level_11_Demo_Verify.TC_02_Login(Level_11_Demo_Verify.java:91)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br />
              sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)<br />
              sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)<br />
              java.lang.reflect.Method.invoke(Unknown Source)<br />
              org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)<br />
              org.testng.internal.Invoker.invokeMethod(Invoker.java:583)<br />
              org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)<br />
              org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)<br />
              org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)<br />
              org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)<br />
              org.testng.TestRunner.privateRun(TestRunner.java:648)<br />
              org.testng.TestRunner.run(TestRunner.java:505)<br />
              org.testng.SuiteRunner.runTest(SuiteRunner.java:455)<br />
              org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)<br />
              org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)<br />
              org.testng.SuiteRunner.run(SuiteRunner.java:364)<br />
              org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)<br />
              org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)<br />
              org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)<br />
              org.testng.TestNG.runSuitesLocally(TestNG.java:1137)<br />
              org.testng.TestNG.runSuites(TestNG.java:1049)<br />
              org.testng.TestNG.run(TestNG.java:1017)<br />
              org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)<br />
              org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)<br />
              org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)<br />
                </div>
      </td>
</tr>
      </table>


  
</body>
</html>
